extensions/ql-vscode/src/view/results/raw-results-table.tsx

```diff
@@ -58,4 +76,53 @@ export class RawTable extends React.Component<RawTableProps, Record<string, neve
       </tbody>
     </table>;
   }
+
+  private handleNavigationEvent(event: NavigationEvent) {
+    switch (event.t) {
+      case 'navigateAlert': {
+        this.setState(prevState => {
+          const numberOfAlerts = this.props.resultSet.rows.length;
+          if (numberOfAlerts === 0) {
+            return prevState;
+          }
+          const currentRow = prevState.selectedItem?.row;
+          const nextRow = currentRow === undefined
+            ? 0
+            : (currentRow + event.direction);
```

Comment:
The direction _happens_ to be a number that tells you how far to move in a given direction. I'd rather this be explicit. Either use an enum for direction (as I suggest above) and convert that into 1  / -1, or explicitly use the term `distance` and make it clear that you can accept a number.

Reply:
After the change to up/down/left/right navigation this is more explicit